doc/benchmark/modules/ROOT/pages/scenario/authorization-code.adoc

```diff
@@ -38,7 +38,7 @@ bin/kcb.sh \
   --server-url=http://localhost:8080/ \
   --realm-name=realm-0 \
   --username=user-0 \
-  --user-password=user-password-0 \
+  --user-password=user-0-password \
   --client-id=gatling \
   --client-secret=setup-for-benchmark \
   --client-redirect-uri=http://localhost:8081 \
```

Comment:
Hi @kami619 , please can you change it to localhost:8080, currently it's just misleading.

Reply:
That is true, I intend to fix this to reflect the latest changes from the `initialize-benchmark-entities.sh` which would also be consistent with the current benchmark expects the client, client secret and the client redirect uri to be looking like. It will be part of my next commit to this PR.

```
bin/kcb.sh \
  --scenario=keycloak.scenario.authentication.AuthorizationCode \
  --server-url=http://localhost:8080/ \
  --realm-name=realm-0 \
  --username=user-0 \
  --user-password=user-0-password \
  --client-id=client-0 \
  --client-secret=client-0-secret \
  --client-redirect-uri=* \
  --log-http-on-failure
```

Reply:
got it, I will adjust that accordingly, thanks @ahus1